ui/src/components/PinPreview.tsx

```diff
@@ -1,41 +1,64 @@
 import * as React from "react";
 import styled from "styled-components";
-import { Card } from '@material-ui/core';
+import { Card } from "@material-ui/core";
 import CardContent from "@material-ui/core/CardContent";
 import CardMedia from "@material-ui/core/CardMedia";
 
-import { CardTitleText, CardDescriptionText, CardDetailText } from "../styles/typography";
+import {
+  CardTitleText,
+  CardDescriptionText,
+  CardDetailText,
+  CardTagText,
+} from "../styles/typography";
 
 const StyledCard = styled(Card)`
   width: 392px;
-  height: 448px;
+  height: 467px;
   box-shadow: 0px 0px 25px 5px rgba(0, 0, 0, 0.2);
-`
+`;
 
 const StyledMedia = styled(CardMedia)`
   height: 211px;
+`;
+
+const StyledFlag = styled.img`
 `
 
 interface PinPreviewProps {
-  title: string,
-  description: string,
-  author: string,
-  date: string,
+  title: string;
+  description: string;
+  author: string;
+  date: string;
+  country: string;
 }
 
-export function PinPreview({title, description, author, date}: PinPreviewProps) : JSX.Element {
+declare let require: any;
+
+export function PinPreview({
+  title,
+  description,
+  author,
+  date,
+  country,
+}: PinPreviewProps): JSX.Element {
 
   return (
     <StyledCard>
-      <StyledMedia/>
+      <StyledMedia 
+        image={require('../assets/images/temp.png')}
```

Comment:
Don't think we need `require()` here, you should just be able to do `import image from "../assets/images/temp.png"` and supply that to the prop.

We shouldnt be using `commonJS` require imports in this code base (we can remove the node types and typescript changes then too)

Reply:
Yeah Hanlin and I wanted some help with this. TS wasn't letting us directly import nor pass the path to the prop. We added the require as a temp working solution